src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/magical/talismans/MagicianTalisman.java

```diff
@@ -40,12 +41,17 @@ public MagicianTalisman(SlimefunItemStack item, ItemStack[] recipe) {
         addItemSetting(allowEnchantmentBooks);
 
         for (Enchantment enchantment : Enchantment.values()) {
-            try {
-                for (int i = 1; i <= enchantment.getMaxLevel(); i++) {
-                    enchantments.add(new TalismanEnchantment(enchantment, i));
+            // Stop custom enchant plugins appearing
+            // We also want to stop some stupid plugins which register as Minecraft enchants
+            // but also go above the max enchant level.
```

Comment:
multi-line comments :eyes:
```suggestion
            /* 
            * Stop custom enchant plugins appearing
            * We also want to stop some stupid plugins which register as Minecraft enchants
            * but also go above the max enchant level.
            */
```